tests/publiccloud/prepare_tools.pm

```diff
@@ -103,10 +108,13 @@ sub run {
     assert_script_run("img-proof list");
     record_info('img-proof', script_output('img-proof --version'));
 
-    # Install Terraform from repo
-    zypper_call('ar https://download.opensuse.org/repositories/systemsmanagement:/terraform/SLE_15_SP2/systemsmanagement:terraform.repo');
-    zypper_call('--gpg-auto-import-keys -q in terraform');
-    record_info('Terraform', script_output('terraform -v'));
+    my $terraform_wrapper = <<EOT;
+#!/bin/sh
+podman run -i -t hashicorp/terraform:0.14.1 \$1
```

Comment:
what about make the version a variable? isnt going to change in some point?

Reply:
yeah I thought about that too , now when you asking I will do this :)

Reply:
but lets use some version as default, and the variable only for debugging...

Reply:
mmm wait are you speaking about openQA variable ? don't like this idea . please keep in mind use case . it is not test which you run on daily basis . and than one day you want to just do single run to check something . This test suit is logic used for generation of image used by **all** public cloud test . so whenever we would need to bump up terraform version I would like to see dedicated commit in source code , not just dummy clone job with `TERRAFORM_VER='15.1'`

Reply:
Actually, Clemens' idea is good. We could add the possibility to change quickly the version with variable for testing, but hardcode the default version here.

Reply:
I think my main fear here that someone will clone the job and bump up terraform version without commit message than. Also again keeping in mind the context (this job is not running everyday ) I don't see real need of such variable ...